.dockerignore

```diff
@@ -14,3 +14,4 @@ junit.xml
 npm-debug.log
 static-analysis
 setupEnzyme.js
+node_modules
```

Comment:
I added this because we need *docker* to handle building in order to allow docker-compose to inject environment variables, which are fixed at build time.